### DISCUSSION TITLE
Reload extension information

### DISCUSSION THREAD

**Discussion by komir2004** [akeeba/panopticon#321] [Q&A (Community Support)] [answered]
Hi, well notice one problem. For some web pages, extension information is not "reloaded" automatic, I assume it is run by cron.
When run manually reload, Panopticon find updates.
I have cron job and it is fired every time without any interruption
`Akeeba Panopticon CLI 1.0.4
Copyright (c) 2023-2023  Akeeba Ltd

Distributed under the terms of the GNU General Public License as published
by the Free Software Foundation, either version 3 of the License, or (at your
option) any later version. See LICENSE.txt.

Runs scheduled tasks
====================

[info] Locking task tables
[info] Cleaning up stuck tasks
[info] Getting next task
[info] System Task #24 - “Schedule Automatic Joomla Update”
[info] Task finished with status “OK”
[info] Locking task tables
[info] Cleaning up stuck tasks
[info] Getting next task
[info] System Task #26 - “Send Enqueued Email Messages”
[info] Task finished with status “OK”
[info] Locking task tables
[info] Cleaning up stuck tasks
[info] Getting next task
[info] There are no pending tasks.
[info] Locking task tables
[info] Cleaning up stuck tasks
[info] Getting next task
[info] There are no pending tasks.
[info] Locking task tables
[info] Cleaning up stuck tasks
[info] Getting next task
[info] There are no pending tasks.
[info] Locking task tables
[info] Cleaning up stuck tasks
[info] Getting next task
[info] There are no pending tasks.
[info] Locking task tables
[info] Cleaning up stuck tasks
[info] Getting next task
[info] There are no pending tasks.
[info] Locking task tables
[info] Cleaning up stuck tasks
[info] Getting next task
[info] There are no pending tasks.
[info] Locking task tables
[info] Cleaning up stuck tasks
[info] Getting next task
[info] There are no pending tasks.
[info] Locking task tables
[info] Cleaning up stuck tasks
[info] Getting next task
[info] There are no pending tasks.
[info] Locking task tables
[info] Cleaning up stuck tasks
[info] Getting next task
[info] There are no pending tasks.
[info] Locking task tables
[info] Cleaning up stuck tasks
[info] Getting next task
[info] There are no pending tasks.
[info] Locking task tables
[info] Cleaning up stuck tasks
[info] Getting next task
[info] There are no pending tasks.
[info] Locking task tables
[info] Cleaning up stuck tasks
[info] Getting next task
[info] There are no pending tasks.
[info] Locking task tables
[info] Cleaning up stuck tasks
[info] Getting next task
[info] System Task #26 - “Send Enqueued Email Messages”
[info] Task finished with status “OK”
[info] Locking task tables
[info] Cleaning up stuck tasks
[info] Getting next task
[info] There are no pending tasks.
[info] Locking task tables
[info] Cleaning up stuck tasks
[info] Getting next task
[info] There are no pending tasks.
[info] Locking task tables
[info] Cleaning up stuck tasks
[info] Getting next task
[info] There are no pending tasks.
[info] Locking task tables
[info] Cleaning up stuck tasks
[info] Getting next task
[info] There are no pending tasks.
[info] Locking task tables
[info] Cleaning up stuck tasks
[info] Getting next task
[info] There are no pending tasks.
[info] Locking task tables
[info] Cleaning up stuck tasks
[info] Getting next task
[info] There are no pending tasks.
[info] Locking task tables
[info] Cleaning up stuck tasks
[info] Getting next task
[info] There are no pending tasks.
[info] Locking task tables
[info] Cleaning up stuck tasks
[info] Getting next task
[info] There are no pending tasks.
[info] Locking task tables
[info] Cleaning up stuck tasks
[info] Getting next task
[info] There are no pending tasks.
[info] Locking task tables
[info] Cleaning up stuck tasks
[info] Getting next task
[info] There are no pending tasks.
[info] Locking task tables
[info] Cleaning up stuck tasks
[info] Getting next task
[info] There are no pending tasks.
[info] Locking task tables
[info] Cleaning up stuck tasks
[info] Getting next task
[info] There are no pending tasks.
[info] Locking task tables
[info] Cleaning up stuck tasks
[info] Getting next task
[info] System Task #26 - “Send Enqueued Email Messages”
[info] Task finished with status “OK”
[info] Locking task tables
[info] Cleaning up stuck tasks
[info] Getting next task
[info] There are no pending tasks.
[info] Locking task tables
[info] Cleaning up stuck tasks
[info] Getting next task
[info] There are no pending tasks.
[info] Locking task tables
[info] Cleaning up stuck tasks
[info] Getting next task
[info] There are no pending tasks.
`

**Comment by nikosdion** [comment, marked as answer]
Please read https://github.com/akeeba/panopticon/wiki/Periodic-information-update
